setup.py

```diff
@@ -232,6 +206,14 @@ def cython_extensions():
     'sphinx_rtd_theme>=0.1.8',
     'six>=1.10',
 )
+if BUILD_WITH_CYTHON:
+  sys.stderr.write(
+    "You requested a Cython build via GRPC_PYTHON_BUILD_WITH_CYTHON, "
+    "but do not have Cython installed. We won't stop you from using "
+    "other commands, but the extension files will fail to build.")
+elif NEED_CYTHON:
+  sys.stderr.write('We could not find Cython. Setup could take a while...\n')
```

Comment:
Minutes or hours? Let's try include something likely to be realistic in this message?